packages/frontend/i18n/src/i18n.gen.ts

```diff
@@ -0,0 +1,7180 @@
+// @ts-nocheck
```

Comment:
seems should be ignore

Reply:
the i18n codegen will not be required on installing